packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -85,7 +85,11 @@ export const DevServerMixin = {
      * @private
      */
     _setupLogging(app) {
-        app.use(expressLogging('dev'))
+        app.use(
+            expressLogging(
+                ':method :url :status :res[content-length] - :response-time ms (:req[x-correlation-id])'
```

Comment:
I believe that you mentioned that we'll be changing the header key name from `x-correlation-id` to `correlation-id`. You'll want to reflect that in this change if that is the case.

Also, can you double check that the format we have now is essentially `dev` + `(:req[x-correlation-id])`. 

The reason I say this is because we don't want to accidentally remove any kind of information from the logs that developers might expect to be there. That being said, there is a potential that some partners will be tailing logs and running them through some sort of automation for their own needs. So if we change the format would that be considered a breaking change? (ideally tacking on the correlation-id at the end won't break most parsing). I'm not super concerned with this tho. Just something I wanted to ask.

Reply:
I confirm that we can use `expressLogging('dev (:req[x-correlation-id])')`